dev-libs/hyperscan/hyperscan-5.2.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake python-r1 flag-o-matic
+
+DESCRIPTION="High-performance regular expression matching library"
+SRC_URI="https://github.com/01org/hyperscan/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+HOMEPAGE="https://www.hyperscan.io/"
```

Comment:
Please also add the Github page as secondary homepage here.

Reply:
done.